pom.xml

```diff
@@ -35,6 +35,12 @@
 			<artifactId>spring-boot-starter-web</artifactId>
 		</dependency>
 
+
+		<dependency>
```

Comment:
I had to nuke my environment because the annotation this brings in couldn't be found (RepositoryRestResource)

Reply:
same, i had to nuke mine too